client/state/selectors/get-checklist-task-urls.js

```diff
@@ -45,12 +47,16 @@ export default createSelector(
 			siteId,
 			getSiteOption( state, siteId, 'page_on_front' )
 		);
+		const updateHomepageUrl =
+			abtest( 'checklistUpdateHomepage' ) === 'templateSelector'
```

Comment:
Why are we gating this based on the A/B test? Wouldn't it make more sense to always enable it when FSE is active for the site (or more broadly for template-first themes)? I don't think this falls into the category of tasks that we'll have to provide stats support in order to launch it for all users. cc @apeatling

Reply:
Agreed, this should be shipped, we don't need an AB test and we can monitor how it is received in the next user test.

Reply:
> Why are we gating this based on the A/B test?

I _think_ the intention was to have it behind a "0 / 100" AB Test config so we can enable it for user testing prior to it going live (as in, if the experience isn't fully refined).

@adamziel, thoughts?

Reply:
Could we just enable it on the staging + horizon environment?

Reply:
Staging + Horizon makes sense to me @jblz @apeatling